proto/chorus/chorus.proto

```diff
@@ -126,11 +126,12 @@ message Replication{
   int64 events = 12;
   int64 events_done = 13;
 
-  google.protobuf.Timestamp last_emitted_at = 14;
-  google.protobuf.Timestamp last_processed_at = 15;
+  google.protobuf.Timestamp init_done_at = 15;
```

Comment:
changing field order will break proto backward compatibility. 
New fields should be appended to the end with increased number.